webservices/common/models/reports.py

```diff
@@ -333,6 +337,8 @@ class BaseFiling(PdfMixin, CsvFecMixin, db.Model):
     election_state = db.Column('el_state', db.String)
     receipt_date = db.Column('create_dt', db.Date, index=True)
     sign_date = db.Column(db.Date)
+    means_filed = 'e-file'
```

Comment:
Well, circling back to this now I think a better thing to do would be yank them entirely, since they are serving no purpose (no filtering or anything).  And it's discernible from the endpoin itself that these resources were filed electronically.